backend/backend/scoring.py

```diff
@@ -0,0 +1,124 @@
+import typing
+
+
+# these are the rotios we are gonna use and their weights(0-1)
+ratios_weights = {
+    "current_ratio": 0.5,
+    "roic": 0.9,
+    "debt_to_equity": 0.7
+    }
+
+max_score = 0
+
+for key in ratios_weights:
+    max_score += ratios_weights[key]
+
+
+
+def evaluate_current_ratio(value: float) -> float:
+    # input: ratio value
+    # output: a score between 0 and 1, 0.5 is considered okay
+    # 0.4-0.7: 0.3
+    scoring_dic = {
+        float("-inf"): 0,
+        0.1: 0,
+        0.4: 0.1,
+        0.7: 0.3,
+        1: 0.5,
+        1.2: 0.7,
+        1.5: 0.9,
+        float("inf"): 1
+        }
+    score = -1
+    for key in scoring_dic:
+        if key > value:
+            score = scoring_dic[key]
+            break
+    
+    return score
+
+def evaluate_roic(value: float) -> float:
+    # input: ratio value
+    # output: a score between 0 and 1, 0.5 is considered okay
+    scoring_dic = {
+        float("-inf"): 0,
+        2: 0,
+        5: 0.3,
+        10: 0.5,
+        20: 0.7,
+        30: 0.9,
+        float("inf"): 1
+        }
+    score = -1
+    for key in scoring_dic:
+        if key > value:
+            score = scoring_dic[key]
+            break
+    
+    return score
+
+def evaluate_debt_to_equity(value: float) -> float:
+    # input: ratio value
+    # output: a score between 0 and 1, 0.5 is considered okay
+    scoring_dic = {
+        float("-inf"): 1,
+        1: 1,
+        1.5: 0.9,
+        2: 0.7,
+        3: 0.5,
+        5: 0.1,
+        7: 0,
+        float("inf"): 0
+        }
+    score = -1
+    for key in scoring_dic:
+        if key > value:
+            score = scoring_dic[key]
+            break
+    
+    return score
+
+
+
+def scoring(ratios: typing.Dict[str, float]) -> float:
```

Comment:
`from typing import Dict` at start of file
Maybe create a class to represent the ratios? Currently difficult to know what each key should be.